boards/st/nucleo_u575zi_q/nucleo_u575zi_q-common.dtsi

```diff
@@ -176,6 +176,10 @@
 	status = "okay";
 };
 
+&hash {
+	status = "okay";
```

Comment:
Should we enable this by default? Might be a thing to keep off by default to avoid inflating build sizes. Might be okay though, let maintainers say is it or not